stdlib/private/StdlibUnittest/CMakeLists.txt

```diff
@@ -1,31 +1,19 @@
-set(swift_stdlib_unittest_platform_sources)
-set(swift_stdlib_unittest_module_depends
-  SwiftPrivate SwiftPrivatePthreadExtras SwiftPrivateLibcExtras)
-set(swift_stdlib_unittest_framework_depends)
+
 set(swift_stdlib_unittest_compile_flags
   "-Xfrontend" "-disable-objc-attr-requires-foundation-module")
-
 if (SWIFT_RUNTIME_ENABLE_LEAK_CHECKER)
   list(APPEND swift_stdlib_unittest_compile_flags "-DSWIFT_RUNTIME_ENABLE_LEAK_CHECKER")
 endif()
-
-if(SWIFT_HOST_VARIANT MATCHES "${SWIFT_DARWIN_VARIANTS}")
-  list(APPEND swift_stdlib_unittest_platform_sources
-    GetOSVersion.mm)
-  list(APPEND swift_stdlib_unittest_module_depends
-    Darwin
-    Foundation)
-  list(APPEND swift_stdlib_unittest_framework_depends
-    Foundation)
-endif()
-if(SWIFT_HOST_VARIANT STREQUAL "linux" OR SWIFT_HOST_VARIANT STREQUAL "freebsd")
-  list(APPEND swift_stdlib_unittest_module_depends
-    Glibc)
-endif()
 if(SWIFT_SERIALIZE_STDLIB_UNITTEST)
   list(APPEND swift_stdlib_unittest_compile_flags "-Xfrontend" "-sil-serialize-all")
 endif()
 
+# TODO: extend this through a similar mechanism to SWIFT_MODULE_DEPENDS
+set(swift_stdlib_unittest_framework_depends)
+if(SWIFT_HOST_VARIANT MATCHES "${SWIFT_DARWIN_VARIANTS}")
+  list(APPEND swift_stdlib_unittest_framework_depends Foundation)
+endif()
```

Comment:
I'm not sure if this is actually needed, might be an old workaround for something.  StdlibUnittest imports the Foundation overlay, so there should be no need to list Foundation as an explicit dependency.  You could try removing this block in a future PR.